site/content/en/docs/start/_index.md

```diff
@@ -1,8 +1,190 @@
 ---
-title: "Get Started!"
+title: "minikube start"
+linkTitle: "Get Started!"
 weight: 1
-description: >
-  How to install minikube
 aliases:
   - /docs/start
 ---
+
+minikube is local Kubernetes, focusing on making it easy to learn and develop for Kubernetes. 
+
+All you need is Docker (or similarly compatible) container or a Virtual Machine environment, and Kubernetes is a single command away: `minikube start`
+
+
+## What you’ll need
+
+* 2GB of free memory
+* 20GB of free disk space
+* Internet connection
+* Container or virtual machine manager, such as: [Docker]({{<ref "/docs/drivers/docker">}}), [Hyperkit]({{<ref "/docs/drivers/hyperkit">}}), [Hyper-V]({{<ref "/docs/drivers/hyperv">}}), [KVM]({{<ref "/docs/drivers/kvm2">}}), [Parallels]({{<ref "/docs/drivers/parallels">}}), [Podman]({{<ref "/docs/drivers/podman">}}), [VirtualBox]({{<ref "/docs/drivers/virtualbox">}}), or [VMWare]({{<ref "/docs/drivers/vmware">}})
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
+
+{{% tabs %}}
+{{% tab "Linux" %}}
+
+For Linux users, we provide 3 easy download options:
+
+### Binary download
+
+```shell
+ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+ sudo install minikube-linux-amd64 /usr/local/bin/minikube
+```
+
+### Debian package
+
+```shell
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_{{< latest >}}-0_amd64.deb
+sudo dpkg -i minikube_{{< latest >}}-0_amd64.deb
+```
+
+### RPM package
+
+```shell
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-{{< latest >}}-0.x86_64.rpm
+sudo rpm -ivh minikube-{{< latest >}}-0.x86_64.rpm
+```
+
+{{% /tab %}}
+{{% tab "macOS" %}}
+
+If the [Brew Package Manager](https://brew.sh/) installed:
+
+```shell
+brew install minikube
+```
+
+Otherwise, download minikube directly:
+
+```shell
+curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64
+sudo install minikube-darwin-amd64 /usr/local/bin/minikube
+```
+
+{{% /tab %}}
+{{% tab "Windows" %}}
+
+If the [Chocolatey Package Manager](https://chocolatey.org/) is installed, use it to install minikube:
+
+```shell
+choco install minikube
+```
+
+Otherwise, download and run the [Windows installer](https://storage.googleapis.com/minikube/releases/latest/minikube-installer.exe)
+
+{{% /tab %}}
+{{% /tabs %}}
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">2</strong></span>Start your cluster</h2>
```

Comment:
Thanks for adding that beautiful Gui Step Number, I look forward to see follow up PRs to make all the Documentation look like this for Numbered items . for example Push image docs.
(we should make an issue for someone to pick up this task) this Graphical Number Icon adds a lot of usability to for ppl who dont like to read docs

Reply:
Thanks!